app/authentication/map_role_to_permissions.py

```diff
@@ -0,0 +1,12 @@
+from app import settings
+
+
+def map_role_to_permissions(roles):
```

Comment:
This seems like something that would be better suited to living within a authentication helper.